tests/pages/base.py

```diff
@@ -41,3 +46,18 @@ def languages(self):
         def select_language(self, value):
             el = self.root.find_element(*self._language_locator)
             Select(el).select_by_value(value)
+
+    class FamilyNavigation(PageRegion):
+
+        _tertiary_nav_button = (By.ID, 'fxfamilynav-tertiarynav-trigger')
+        _tertiary_nav_menu = (By.ID, 'fxfamilynav-tertiarynav')
+
+        def click_tertiary_menu(self):
+            self.selenium.find_element(*self._tertiary_nav_button).click()
+            Wait(self.selenium, self.timeout).until(
+                expected.visibility_of_element_located(self._tertiary_nav_menu))
+
+        @property
+        def tertiary_menu_visible(self):
+            el = self.selenium.find_element(*self._tertiary_nav_menu)
+            return el.is_displayed()
```

Comment:
I think I would have a function directly under the base page named `open_x_menu` you've used both 'family' and 'tertiary', so I'd pick one and use it consistently. This function could return your `PageRegion` object, with functions and properties related to the menu itself. Rather than returning whether the menu is visible, a better _functional_ test might click one of the menu items and check that you get the expected response. Otherwise, with the mentioned suggestions your test might look something like this:

```python
menu = page.open_tertiary_menu()
assert menu.is_displayed
```